roles/pulp-web/templates/pulp-web.service.yaml.j2

```
@@ -47,7 +47,8 @@ spec:
 {% endif %}
 {% if ingress_type | lower == "loadbalancer" %}
   type: LoadBalancer
-{% elif ingress_type != "none" %}
+{% elif ingress_type|lower == "none" %}
+      nodePort: 24880
```

Actually, changing this made me able to remove `content_origin` from the CRD

---

I would think this would be:
```suggestion
{% elif ingress_type|lower == "nodeport" %}
      nodePort: 24880
```
https://github.com/pulp/pulp-operator/blob/main/deploy/crds/pulpproject_v1beta1_pulp_crd.yaml#L159

---

thank you!